classification1.Rmd

```diff
@@ -206,28 +200,28 @@ changing them to "Benign" and "Malignant" using the `labels` argument.
 ```{r 05-scatter, fig.height = 4, fig.width = 5, fig.cap= "Scatter plot of concavity versus perimeter coloured by diagnosis label"}
 perim_concav <- cancer %>%
   ggplot(aes(x = Perimeter, y = Concavity, color = Class)) +
-  geom_point(alpha = 0.5) +
-  labs(color = "Diagnosis") +
+  geom_point(alpha = 0.6) +
+  labs(color = "Diagnosis", x = "Perimeter (standardized)", y = "Concavity (standardized)") +
   scale_color_manual(labels = c("Malignant", "Benign"), values = c("orange2", "steelblue2"))
 perim_concav
 ```
 
 In Figure \@ref(fig:05-scatter), we can see that malignant observations typically fall in
-the the upper right-hand corner of the plot area. By contrast, benign
-observations typically fall in lower left-hand corner of the plot. In other words, benign observations 
-tend to have lower concavity and perimeter values, and malignant ones tends to have larger 
-concavity and perimeter values. Suppose we
+the upper right-hand corner of the plot area. By contrast, benign
+observations typically fall in the lower left-hand corner of the plot. In other words,
+benign observations tend to have lower concavity and perimeter values, and malignant
+ones tend to have larger values. Suppose we
 obtain a new observation not in the current data set that has all the variables
 measured *except* the label (i.e., an image without the physician's diagnosis
-for the tumour class). We could compute the perimeter and concavity values,
+for the tumor class). We could compute the standardized perimeter and concavity values,
 resulting in values of, say, 1 and 1. Could we use this information to classify
 that observation as benign or malignant? Based on the scatter plot, how might 
 you classify that new observation? How would you classify a new observation with
-perimeter value of -1 and concavity value of -0.5? What about 0 and 1? It seems
-like the *prediction of an unobserved label* might be possible, based on our
-visualization. 
+a standardized perimeter value of -1 and a concavity value of -0.5? What about 0 and 1? 
+Based on our visualization, it seems
+like the *prediction of an unobserved label* might be possible.
 
-## Classification with K-nearest neighbours
+## Classification with $K$-nearest neighbors
```

Comment:
I agree with this change in general. But I have noticed in the HTML navigation table that appears on the left, the $K$ symbol is really, really small.

I am not sure this is an issue we should fix, since it seems to be a problem with how rbookdown renders math in the navigation pane.

Reply:
referenced in new issue #240